cli/command_maintenance_info.go

```diff
@@ -25,7 +25,16 @@ func runMaintenanceInfoCommand(ctx context.Context, rep repo.DirectRepository) e
 	}
 
 	if jsonOutput {
-		printStdout("%s\n", jsonBytes(s))
+		mi := struct {
```

Comment:
can we define a named structure here and add an e2e test that parses it?

Reply:
addressed feedback